# create_python_driver

Having trouble with permission for your serial cable? (/dev/ttyUSB0)

Use:  sudo chmod a+rw /dev/ttyUSB0

for a temporary fix

Or:   sudo adduser {username_here} dailout

for a more permenant fix
